stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1082,6 +1086,17 @@ internal constructor(
         disconnect(true).execute()
     }
 
+    /**
+     * Cancel all jobs on the ChatClient Scope.
+     * This method postpone a new coroutine to cancel all children jobs.
+     */
+    private fun postponeCancelScope() {
+        scope.launch {
+            delay(100)
+            scope.coroutineContext.cancelChildren()
+        }
+    }
```

Comment:
We need to schedule and delay this process to be able to pass our tests because in another case the test itself is canceled, and not pass our test suite